near-plugins/tests/contracts/pausable/rust-toolchain

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.64.0"
```

Comment:
Could probably use 1.65.0 if we wanted to.

Reply:
Updating to `1.65.0` works fine locally on my machine, however it makes [CI fail](https://github.com/aurora-is-near/near-plugins/actions/runs/3747724193/jobs/6364726907). Perhaps CI cannot handle differing channels in [root](https://github.com/aurora-is-near/near-plugins/blob/b679c29fb45698d53cfed8abcc37a64664c57907/rust-toolchain) and test contract toolchains.

I openend #52 and would leave the channel as is for this PR.